client/my-sites/site-settings/administration/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import isSiteWpcomStaging from 'calypso/state/selectors/is-site-wpcom-staging';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import SiteTools from '../site-tools';
+import { SOURCE_SETTINGS_ADMINISTRATION } from '../site-tools/utils';
+
+export default function AdministrationSettings() {
+	const isWpcomStagingSite = useSelector( ( state ) =>
```

Comment:
I wonder if we still need this guard here if we already hide the subtab via the `enabled` predicate 🤔

Reply:
not really, was 50-50 whether it was worth keeping.

Reply:
Just for a decision, leaving it in as it makes it a bit more self contained, I think you can still access the route via url or site selector.

Reply:
I'll add some copy for when it's a staging site even though I don't think anyone would see it. Or maybe that's what we should display always and not hide the menu?